rules/use-https-text/rule.md

```diff
@@ -12,14 +12,18 @@ created: 2022-11-30T00:13:40.254Z
 guid: 4bf87ccf-91a1-470a-b1d6-bb2363761e06
 ---
 
-Have you ever seen the text in an email or YouTube description meant to be a link to a page and wondered why it's missing the customary blue underline or can't be clicked?
+Sometimes, you may come across text in an YouTube description that appears to be a link but lacks the familiar blue color and underline and isn't clickable. This often happens because the text is missing the "https://" prefix.
```

Comment:
```suggestion
Sometimes, you may come across text in a YouTube description that appears to be a link but lacks the familiar blue color plus underline and isn't clickable. This often happens because the text is missing the "https://" prefix.
```